tests/docker/ducker-ak

```diff
@@ -58,10 +58,14 @@ help|-h|--help
     Display this help message
 
 up [-n|--num-nodes NUM_NODES] [-f|--force] [docker-image]
+        [-E|--custom-ducktape DIR]
     Bring up a cluster with the specified amount of nodes (defaults to ${default_num_nodes}).
     The docker image name defaults to ${default_image_name}.  If --force is specified, we will
     attempt to bring up an image even some parameters are not valid.
 
+    If --custom-ducktape is specified, we will install the provided custom
```

Comment:
Might want to clarify that this is the ducktape repo, not, e.g., the actual installed module.